core/App/screens/Splash.tsx

```diff
@@ -141,29 +142,33 @@ const Splash: React.FC<SplashProps> = (props: SplashProps) => {
           return
         }
 
-        const newAgent = new Agent(
-          {
-            label: 'Aries Bifold',
-            mediatorConnectionsInvite: Config.MEDIATOR_URL,
-            mediatorPickupStrategy: MediatorPickupStrategy.Implicit,
-            walletConfig: { id: credentials.id, key: credentials.key },
-            autoAcceptConnections: true,
-            autoAcceptCredentials: AutoAcceptCredential.ContentApproved,
-            logger: new ConsoleLogger(LogLevel.trace),
-            indyLedgers,
-            connectToIndyLedgersOnStartup: true,
-            autoUpdateStorageOnStartup: true,
-          },
-          agentDependencies
-        )
-
-        const wsTransport = new WsOutboundTransport()
-        const httpTransport = new HttpOutboundTransport()
-
-        newAgent.registerOutboundTransport(wsTransport)
-        newAgent.registerOutboundTransport(httpTransport)
-
-        await newAgent.initialize()
+        let newAgent = agent
```

Comment:
Does this do anything? I think objects are passed by ref so I'm not sure it will mint a new object.